ilastik/applets/tracking/conservation/conservationTrackingGui.py

```diff
@@ -96,25 +123,35 @@ def _loadUiFile(self):
         return self._drawer
 
     @staticmethod
-    def getAvailablePgmlinkSolverTypes():
+    def getAvailableTrackingSolverTypes():
         solvers = []
-        if hasattr(pgmlink.ConsTrackingSolverType, "CplexSolver"):
-            solvers.append("ILP")
-
-        if hasattr(pgmlink.ConsTrackingSolverType, "DynProgSolver"):
-            solvers.append("Magnusson")
-
-        if hasattr(pgmlink.ConsTrackingSolverType, "FlowSolver"):
-            solvers.append("Flow")
+        if WITH_HYTRA:
+            try:
+                if dpct:
+                    solvers.append('Flow-based')
+            except:
+                pass
+            try:
+                if mht:
+                    solvers.append('ILP')
+            except:
+                pass
+        else:
```

Comment:
Writing `except: pass` is always super-sketchy.  It's best to catch the specific error you're expecting to see.